man/halt.xml

```diff
@@ -163,13 +163,6 @@
     otherwise.</para>
   </refsect1>
 
-  <refsect1>
-    <title>Notes</title>
-
-    <para>These are legacy commands available for compatibility
-    only.</para>
```

Comment:
hmm, i think the commands are not actually equivalent in regards to polkit hookup. we spawn the agent for systemctl poweroff, but not for poweroff (for compat reasons).

I'd continue to push people towards the new commands really. I am fine with toning things down a bit, but it's not that they were entirely equivalent and it wasn't preferable to user the new commands...

Reply:
Maybe we can just add polkit support to reboot/halt/poweroff? The only "use case" that'd be broken is that somebody is issuing reboot commands and counting on them failing because of lack of privileges, but I don't think that's a legitimate concern.

Reply:
the thing is that the polkit stuff might just result in hanging where people want clean and actionable failure. think distro update script that reboots as last step, and somebody fucked up the selinux or policykit policy, and now all the systems just hang when the adminds would much rather be notified about such policy fuckups right-away... I always was too shy to change behaviour in this regard, in particular as, well, polkit has probably an acceptance problem for sysv enthusiasts who really love the old commands...